todo_app/tests_e2e/test_selenium.py

```diff
@@ -0,0 +1,141 @@
+import os
+import pytest
+import requests
+
+from todo_app import app, data
+from threading import Thread
+
+from dotenv import load_dotenv, find_dotenv
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+BOARD_URL = "https://api.trello.com/1/boards/"
+BOARDS_URL = "https://api.trello.com/1/members/me/boards?fields=name,url"
+
+file_path = find_dotenv('.env')
+load_dotenv(file_path, override=True)
```

Comment:
Putting this here instead of inside `test_app()` means that you cannot run `poetry run pytest`. Because the integration test runs first and overwrites the env vars.

Reply:
I did try this but it broke the tests in this file so moved it back till I work out the best way to do it